app/src/main/java/com/globant/practice/domain/model/Profile.java

```diff
@@ -0,0 +1,110 @@
+package com.globant.practice.domain.model;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Represents the user profile.
+ * Created by jonathan.vargas on 5/04/2017.
+ */
+
+public class Profile {
+    @SerializedName("login")
```

Comment:
You do not need a serialized name when the attribute name is the same. Apply this in all cases